deliveries/lastic-milestone-3-4.md

```diff
@@ -0,0 +1,219 @@
+# Milestone Delivery :mailbox:
+
+**Submission Reminder:**
+> Only the GitHub account that submitted the original application is permitted to submit milestones. Please adhere to this rule to ensure a smooth review process.
+
+**Mandatory Guidelines:**
+> It is crucial to follow the [official milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md) closely. Do not remove any sections marked as mandatory or outlined with bold letters or headlines. Optional advisory comments, like this one, can be removed for clarity.
+
+* **Application Document:** [lastic.md](https://github.com/w3f/Grants-Program/blob/master/applications/Lastic.md)
```

Comment:
```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/Lastic.md
```

I think this syntax messes with our GitHub actions.

Reply:
Sorry, the link was incorrect I was my mistake.